Client/Frontend/Widgets/SimpleHighlightCell.swift

```diff
@@ -0,0 +1,191 @@
+        contentView.addSubview(selectedOverlay)
+        contentView.addSubview(titleLabel)
+        contentView.addSubview(timeStamp)
+        contentView.addSubview(statusIcon)
+
+        siteImageView.snp_remakeConstraints { make in
+            make.top.equalTo(contentView).offset(SimpleHighlightCellUX.CellTopBottomOffset)
+            make.bottom.equalTo(contentView).offset(-SimpleHighlightCellUX.CellTopBottomOffset).priorityLow()
+            make.leading.equalTo(contentView).offset(SimpleHighlightCellUX.CellSideOffset)
+            make.size.equalTo(SimpleHighlightCellUX.SiteImageViewSize)
+        }
+
+        selectedOverlay.snp_remakeConstraints { make in
+            make.edges.equalTo(contentView)
+        }
+
+        titleLabel.snp_remakeConstraints { make in
+            make.leading.equalTo(siteImageView.snp_trailing).offset(SimpleHighlightCellUX.CellTopBottomOffset)
+            make.trailing.equalTo(contentView).inset(SimpleHighlightCellUX.CellSideOffset)
+            make.top.equalTo(siteImageView)
+        }
+
+        descriptionLabel.snp_remakeConstraints { make in
+            make.leading.equalTo(statusIcon.snp_trailing).offset(SimpleHighlightCellUX.TitleLabelOffset)
+            make.bottom.equalTo(statusIcon)
+        }
+
+        timeStamp.snp_remakeConstraints { make in
+            make.trailing.equalTo(contentView).inset(SimpleHighlightCellUX.CellSideOffset)
+            make.bottom.equalTo(descriptionLabel)
+        }
+
+        statusIcon.snp_remakeConstraints { make in
+            make.leading.equalTo(titleLabel)
+            make.top.equalTo(titleLabel.snp_bottom).offset(SimpleHighlightCellUX.CellTopBottomOffset)
+            make.size.equalTo(SimpleHighlightCellUX.StatusIconSize)
+            make.bottom.equalTo(contentView).offset(-SimpleHighlightCellUX.AlternateBottomOffset).priorityHigh()
+        }
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    func setImageWithURL(url: NSURL) {
+        siteImageView.sd_setImageWithURL(url) { (img, err, type, url) -> Void in
+            guard let img = img else {
+                return
+            }
+            self.siteImage = img
+        }
+        siteImageView.layer.masksToBounds = true
+    }
+
+    func configureSimpleHighlightCell(site: Site) {
+        if let icon = site.icon {
+            let url = icon.url
+            self.siteImageView.layer.borderWidth = 0
+            self.setImageWithURL(NSURL(string: url)!)
+        } else if let url = NSURL(string: site.url) {
+            self.siteImage = FaviconFetcher.getDefaultFavicon(url)
+            self.siteImageView.layer.borderWidth = SimpleHighlightCellUX.BorderWidth
+        }
+        self.titleLabel.text = site.title.characters.count <= 1 ? site.url : site.title
+        configureCellStatus(site)
+        self.timeStamp.text = NSDate.fromMicrosecondTimestamp((site.latestVisit?.date)!).toRelativeTimeConciseString()
```

Comment:
The explicit unwrap after a condition unwrap makes me nervous. I would `guard` this instead of `!`. The default state can be a blank string if it doesn't exist.